Misc/NEWS.d/next/Library/2019-05-30-15-51-42.bpo-28724.34TrS8.rst

```diff
@@ -0,0 +1,2 @@
+The socket module now has the :meth:`socket.send_fds` and :meth:`socket.recv.fds` methods.

```

Comment:
There are functions. I prefer to be specific, since socket.socket is commonly written just as "socket".